pkg/apis/componentconfig/types.go

```diff
@@ -503,6 +503,8 @@ type KubeControllerManagerConfiguration struct {
 	// horizontalPodAutoscalerSyncPeriod is the period for syncing the number of
 	// pods in horizontal pod autoscaler.
 	HorizontalPodAutoscalerSyncPeriod unversioned.Duration `json:"horizontalPodAutoscalerSyncPeriod"`
+	// URL to an external heapster backend
+	Heapster string `json:"heapster"`
```

Comment:
Oh! I meant I had HeapsterUrl locally and then changed it, so totally my bad ;)

If we all agree Im more than happy to make it HeapsterUrl.